components/sup/src/main.rs

```diff
@@ -115,15 +116,27 @@ fn start() -> Result<()> {
             process::exit(ERR_NO_RETRY_EXCODE);
         }
     };
+
+// I'm not sure this is the best way to do this - but it beats having this in every sub_ func
+    match app_matches.subcommand() {
+      (_, Some(m)) => {
+
+        if m.is_present("VERBOSE") {
+            hcore::output::set_verbose(true);
+        }
+      }
+      _ => unreachable!(),
+    } 
+
```

Comment:
Previously, each `sub_` function checked for `VERBOSE` and `NO_COLOR`, but if I'm reading correctly, now they'll only check for `VERBOSE`. It looks like only `sub_run` now checks for `NO_COLOR`, and it is also the only one that checks `JSON`. 

Was this intentional?

Reply:
This is intentional. 

My thinking is that VERBOSE can  be set on anything, so I pulled it up above the `sub_` calls, so we only need it once.

`sub_run` is the only one which gets `no_color` and `json` as these are really log style formatters for a headless mode - which is what `hab sup run` is.

Currently there is no way to even make `hab sup start` etc... colourless and/or json as it mixes UI calls with logging calls on stdout.